Sources/Tools/MaxComponent/plAutoUIBase.cpp

```diff
@@ -500,17 +512,7 @@ void plAutoUIBase::ICreateControls()
 
     RECT rect;
     GetWindowRect(fhDlg, &rect);
-
-    // This used to use MoveWindow() to resize the rollup, but in Max 2022,
-    // that does not seem to work anymore. So, we now do the same thing
-    // that WM_SIZE_PANEL does.
-    IRollupWindow* rollup = GetCOREInterface()->GetCommandPanelRollup();
-    int index = rollup->GetPanelIndex(fhDlg);
-
-    if (index >= 0)
-        rollup->SetPageDlgHeight(index, yOffset + 5);
-
-    InvalidateRect(fhDlg, nullptr, TRUE);
+    MoveWindow(fhDlg, rect.left, rect.top, rect.right - rect.left, yOffset+5, FALSE);
```

Comment:
Potentially breaks Max 2022. We should probably do some version dependent magic here,

Reply:
Yeah, I figured we'd have to.  I didn't want to hold up posting the draft since this appeared on me at the last moment. :)